serving/docker/dockerd-entrypoint-with-cuda-compat.sh

```diff
@@ -22,6 +30,14 @@ else
     echo "Skip CUDA compat libs setup as package not found"
 fi
 
+# Convert TGI Environment Variables to LMI Equivalents
+translateTGIToLMI "MODEL_ID" "HF_MODEL_ID"
```

Comment:
how about we map to `OPTION_MODEL_ID`

Reply:
That makes sense. Looking through the code there are two things that stand out to me:
- We support HF_MODEL_ID and OPTION_MODEL_ID (though not in the same ways). Need to fix that
- We support TENSOR_PARALLEL_DEGREE and OPTION_TENSOR_PARALLEL_DEGREE. Need to consolidate

Reply:
I think we can stick with OPTION_MODEL_ID and also OPTION_TENSOR_PARALLEL_DEGREE. @frankfliu what is your thoughts on this. And gradually remove HF_MODEL_ID and TENSOR_PARALLEL_DEGREE usage in our code base.

Reply:
I think we should keep HF_MODEL_ID, HF_TASK, AND HF_REVISION in our source code. They are not only used for LLM.

Reply:
I don't think we should not encourage `OPTION_*` env var usage, it's more of a workaround. Those are per model settings, and should live if `serving.properties`